koku/cost_models/migrations/0002_auto_20200318_1233.py

```diff
@@ -1,15 +1,18 @@
 # Generated by Django 2.2.11 on 2020-03-18 12:33
```

Comment:
I need to think this through a little more and even try it for myself. But I think you might have to do this metrics loading in a new migration rather than modify an existing one.  I'll get back to you on this.

Reply:
@dccurtis We talked through this. It's a data migration that will break from now on with the model gone. I probably would have taken it a step forward and just had this migration do nothing at all.

Reply:
yeah I just tested this on master, pulled this branch in, and restarted koku.  Worked fine. 👍